src/factories/ModuleFactory.sol

```diff
@@ -7,11 +7,13 @@ import {Ownable2Step} from "@oz/access/Ownable2Step.sol";
 import {Ownable} from "@oz/access/Ownable.sol";
 
 // External Interfaces
-import {IBeacon} from "@oz/proxy/beacon/IBeacon.sol";
+
```

Comment:
Pedantic-Mode Activated: There is an extra line here now :D

Reply:
:+1: